somnia-page-1.html

```diff
@@ -0,0 +1,51 @@
+    <p>Hit the brakes to avoid another opossum and breathe a sigh. (Keep your eyes on the road. Don't watch it run into the underbrush - it will not be alone.) Your fingers shake on the steering wheel as you accelerate again. Be wary. They use the wildlife to their advantage. <i>(Just over the next hill make a split-second choice - do you swerve to avoid the thing in the road that resembles nothing natural? Or do you pretend desperately that you don't see and risk the impact?)</i></p>
+    <p>Be as careful as you can. Watch the lines. Don't turn on your high beams, no matter how other drivers try to convince you. Avoid looking at the woods. And the fields. And even the houses nestled at the edge of the trees. <i>(Do you look behind you to check on the kids? Please let them be the only thing you see in the mirrors when you do.)</i> Breathe.</p>
+    <p>The rustlings followed you farther than you thought. You know that this intersection was safe before; it was safe last night at least. The wind is going the wrong direction for those leaf shadows, for the cloud movement. Don't pay attention. Wipe your hands on your pants - the steering wheel is getting too slippery.</p>
+    <p>Pray you've been convincing.</p>
+    <p>Most of it goes away as the lights of civilization draw closer - except for the feeling between your shoulder blades, that almost-itch that means something is looking at you. <i>(Don't look back, don't look back, don't look back.)</i></p>    <p>Pull in. The kids are still asleep. Get out of the car and look over the lake.</p>
+    <p>The mist over the lake is mostly opaque. The moonlight shines through and makes the weight of it feel like opal and reflects on the lake. It's beautiful.</p>
+    <p>Get the kids into the tent one at a time, nestled under blankets on the same mattress. Turn off the car. Take a breath.</p>
+    <p>That feeling between your shoulders grows.</p>
+    <p>Something in the mists shifts, far out on the water. There's a ripple in the glassy surface, pushing to the shore from the movement above the water.</p>
+    <p>Step back. Enter and zip the tent quickly. Get into the bed, shoes off but jacket on and contacts in and keys still in your pocket. You gather the children close. <i>(They haven't woken through all this and your harsh breathing doesn't even cause them to stir, they're so young and so worn out and sleep-cuddly warm.)</i></p>
+    <p>Don't sleep.</p>
+    <p>Keep your eyes on the road. Don't turn on the high beams. Ignore the shadows playing over the tent walls. It's just lights from other campers. Don't leave empty seats in the car. Pretend you didn't see. Pretend you can't feel the crawly feeling on your skin. Pretend hard.</p>
+    <p>Don't sleep.</p>
+    <p>Don't sleep.</p>
+      
+      
+      </p>
+      <div class="centered-content">
+        <h1>Home Page Directory</h1>
```

Comment:
Instead of re-creating the home page directory here, just provide a link back to the home page.